spec/features/people/list_spec.rb

```diff
@@ -67,4 +67,12 @@
       expect(page).to have_text("Bemerkungen Sektion 1")
     end
   end
+
+  it "allows showing the data_quality column" do
+    visit group_people_path(group_id: group.id)
+    click_link("Spalten")
+    check("Datenqualität")
+    click_link("Spalten")
+    expect(page).to have_css('td[data-attribute-name="data_quality"]')
+  end
```

Comment:
I would suggest adding a spec for all different icons possible and then checking

`expect(find("td[data-attribute-name="data_quality"] i")).to have_css('fas fa-check-circle')`
or
`expect(find("td[data-attribute-name="data_quality"] i")).to have_css('fas fa-check-info')`